modules/ROOT/pages/mobile.adoc

```diff
@@ -33,11 +33,11 @@ include::partial$misc/mobile_platform_compatibility.adoc[]
 
 Add the following `meta` tag to the `head` of pages using {productname} to ensure the mobile user interface functions as intended.
 
-```html+++<meta name="viewport" content="width=device-width, initial-scale=1">++++++</meta>+++
-
+```html
```

Comment:
I'm guessing you'll come back to this later given this PR is focused on links/xrefs only, but this should be using AsciiDoc syntax not markdown syntax.

Reply:
I didn't know we had to change this as well. I just updated my branch with a fix

Reply:
Yes, there are many things that didn't convert properly that need fixing. Things like this, the jekyll markup (`{% if ... %}`), admonitions, etc... all need fixing.